modules/administration/pages/monitoring.adoc

```diff
@@ -254,9 +252,10 @@ image::grafana_sidemenu_datasource.png[scaledwidth=40%]
 +
 image::monitoring_grafana_example.png[scaledwidth=80%]
 
-For more information on how to manually install and configure Grafana, see https://grafana.com/docs.
 
-For more information about the monitoring formulas with forms, see xref:specialized-guides:salt/salt-formula-monitoring.adoc[].
+For more information about the monitoring formulas, see xref:specialized-guides:salt/salt-formula-monitoring.adoc[].
+
+For more information on how to manually install and configure Grafana, see https://grafana.com/docs.
```

Comment:
```suggestion
* For more information about the monitoring formulas, see xref:specialized-guides:salt/salt-formula-monitoring.adoc[].
* For more information on how to manually install and configure Grafana, see https://grafana.com/docs.
```
If there is more than one "for more info" in a row, I'd always use an itemized list.

Reply:
agreed.